runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -203,4 +203,28 @@
 	 */
 	@ScriptArguments(string = 1)
 	public static final int CLAN_SEND_KICK = 215;
+
+	/**
+	 * Builds the items kept on death widget
+	 */
+	@ScriptArguments()
```

Comment:
All of your `@ScriptArguments` are wrong

Reply:
I had thought that Adam meant to leave these empty but he meant to leave them undocumented. I have since updated them and made the script names to be more consistent with other script names used in the class.